packages/sample-app/sample_app/experiment/experiment_example.py

```diff
@@ -0,0 +1,99 @@
+"""
+Example usage of the Experiment context manager
+"""
+
+import asyncio
+import sys
+import os
+
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), ".."))
+
+from openai import OpenAI  # noqa: E402
+from medical_prompts import refuse_medical_advice_prompt, provide_medical_info_prompt  # noqa: E402
+from traceloop.sdk import Traceloop  # noqa: E402
+
+client = Traceloop.init()
+
+
+def generate_medical_answer(prompt_text: str) -> str:
+    """
+    Generate a medical answer using OpenAI and the clinical guidance prompt
+    """
+    openai_client = OpenAI(api_key=os.getenv("OPENAI_API_KEY"))
+
+    response = openai_client.chat.completions.create(
+        model="gpt-3.5-turbo",
+        messages=[{"role": "user", "content": prompt_text}],
+        temperature=0.7,
+        max_tokens=500,
+    )
+
+    return response.choices[0].message.content
+
+
+def medical_task_refuse_advice(row):
+    """Task function for refusing medical advice prompt"""
+    prompt_text = refuse_medical_advice_prompt(row["user-description"])
+    answer = generate_medical_answer(prompt_text)
+    user_description = row["user-description"]
+    print(f"\033[94mMedical user input:\033[0m {user_description}")
+    return {"completion": answer, "prompt": prompt_text}
+
+
+def medical_task_provide_info(row):
+    """Task function for providing medical info prompt"""
+    prompt_text = provide_medical_info_prompt(row["user-description"])
+    answer = generate_medical_answer(prompt_text)
+    user_description = row["user-description"]
+    print(f"\033[94mMedical user input:\033[0m {user_description}")
+    return {"completion": answer, "prompt": prompt_text}
+
+
+def run_experiment_example():
+    """Example using the new run_experiment API to compare two different prompt approaches"""
+
+    print(
+        "\033[95m🔬 Running experiment with clinical guidance prompt (refuses medical advice)...\033[0m"
+    )
+    results_1, errors_1 = asyncio.run(
```

Comment:
nit: make the run experiment method async and put the asyncio.run when calling `run_experiment_example`